.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -7,6 +7,10 @@ labels: sig/cluster-management, kind/feature
 
 <!-- Please use this template if you want to submit a new feature request.-->
 
-**What feature would you like to be added?**
+### User Story
 
-**What are use cases of the feature?**
+### Acceptance criteria
+
+### Current workaround
+
+### Metric to be improved
```

Comment:
What does `Metric to be improved` mean exactly? A comment clarifying it would be very useful.

Reply:
The metrics / KPIs affected by the potential feature. For example: I'd like to tag nodes in providers' platforms when creating them from KKP, because doing this manually costs several hours of human working time weekly. I'd like to improve level of automation, human labour time on operation.
Another example: I'd like to add a provisioning feature to start.kubermatic in the UI instead just providing a file, because there is a high amount of churn between creating the file and starting installation from it. I'd like to decrease churn rate, metric to be improved: conversion rate of installers from start.kubermatic.
```suggestion
### Metric to be improved
<!-- The metrics / KPIs affected by the potential feature. Please describe what metric would you like to increase or decrease with having this feature?
-->
```

Reply:
So when I see this, I would respond with "kubermatic_successful_cluster_reconciliations", a Prometheus metric name, that is surely not what is intended here. How can an external person, or even a developer, know about all our internal product metrics?

I highly doubt you will ever get a meaningful response to this question in the template, even by Kubermatic employees.

Reply:
I understand the intention of this question, but I think we should frame it differently. What about asking something like `What are alternative approaches?`? Users might describe what else we can do or what are they doing right now.

Reply:
As we discussed in the doc, we can reframe it to:

```suggestion
### Use cases
<!-- Please try to explain some use cases why this feature would be valuable for you. If you can, please also explain what metrics would be improved for you with having this feature. -->
```